utils/api.js

```diff
@@ -0,0 +1,28 @@
+export const api = {
+    async call(endpoint, options) {
+        const response = await window.fetch(`api/${endpoint}`, { ...options });
```

Comment:
Le fait de venir faire un spread ici est inutile et coûtera une boucle en plus.

```suggestion
        const response = await window.fetch(`api/${endpoint}`, options);
```